db-engine/src/main/java/com/splicemachine/db/impl/jdbc/authentication/AuthenticationServiceBase.java

```diff
@@ -137,12 +139,38 @@
     */
     protected static final int SECMEC_USRSSBPWD = 8;
 
+	private static final String IMPERSIONATION_ENABLED = "derby.authentication.impersonation.enabled";
+	private static final String IMPERSIONATION_USERS = "derby.authentication.impersonation.users";
+
 	//
 	// constructor
 	//
 	public AuthenticationServiceBase() {
 	}
 
+	protected String impersonate(String userName, String proxyUser) {
+		if (!Boolean.parseBoolean(getProperty(IMPERSIONATION_ENABLED))) {
```

Comment:
It's not really important, but if you want to correct the spelling:  IMPERSIONATION -> IMPERSONATION